### PR TITLE
Adding kafka-init image

### PR DESCRIPTION
**Pull request by kyguy**
(4) Fourth unit to review. Although this isn't exactly complete, (We still need to swap in the "official" jar and checksum), I think it would still be good to review(possibly merge) the scripts. I'll update the Image descriptor file to point to the proper jar once we have them. WDYT?

Issue: https://github.com/jboss-container-images/jboss-amqstreams-1-openshift-image/issues/7

### CODE REVIEW

File: kafka-init/image.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+schema_version: 1
+
+name: redhat-amqstreams-1/amqstreams10-kafkainit-openshift
+description: "AMQ Streams base image for Kafka init"
+version: "1.0"
+from: redhat-amqstreams-1/amqstreams10-javabase-openshift:1.0
```
Comment (ppatierno) [inline kafka-init/image.yaml]: where is the javabase image? I cannot see it in the repo.

File: kafka-init/modules/os-kafka-init-install/install.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+SCRIPTS_DIR=${ADDED_DIR}/scripts
+
+mkdir $KAFKA_HOME
+cp "${SOURCES_DIR}"/*.jar  ${KAFKA_HOME}
+
+chown -R jboss:root ${KAFKA_HOME}
+chmod 0755 ${KAFKA_HOME}
+chmod -R g+rwX ${KAFKA_HOME}
+
+# copy scripts for starting Kafka Connect
```
Comment (ppatierno) [inline kafka-init/modules/os-kafka-init-install/install.sh]: for starting "Kafka Init application" ;)

File: kafka-init/image.yaml (hunk 1 above)
Comment (kyguy) [reply kafka-init/image.yaml]: Just added a PR here: https://github.com/jboss-container-images/jboss-amqstreams-1-openshift-image/pull/18

**Comment by kyguy** [comment]
Updated with official jar and checksum. Ready to go

**Pull request merged**